src/pages/Flow/Settings/Schedules.vue

```diff
@@ -238,7 +239,10 @@ export default {
       }
     },
     timezoneVal(clock) {
-      let tz = this.timezone || Intl.DateTimeFormat().resolvedOptions().timeZone
+      let tz =
+        clock.scheduleType == 'flow'
+          ? 'UTC'
+          : this.timezone || Intl.DateTimeFormat().resolvedOptions().timeZone
```

Comment:
This seems like it'll change the timezone instead of updating the display for the user selected timezone, am I understanding correctly?

Reply:
This method is used in two places, one is the `CronClock` component which is used for abbreviating and the other is the `ClockForm` component for editing. For editing, this method should only preselect the timezone so I don’t think it would change the timezone but maybe I’m missing something? 

I added the `clock.scheduleType == 'flow'` to differentiate between what default should be used as it was defaulting to the user’s settings timezone before

Reply:
Could we make that pre-selected timezone a bit more obvious by showing it in the v-select- such as happens in the user profile?  

What I see in the user profile:

<img width="718" alt="Screen Shot 2021-06-25 at 3 15 17 PM" src="https://user-images.githubusercontent.com/40272060/123474932-b44c3200-d5c8-11eb-85dc-8ed0d491c7b3.png">

What I see in the schedules page (note it does not tell me what the timezone is):

<img width="685" alt="Screen Shot 2021-06-25 at 3 17 46 PM" src="https://user-images.githubusercontent.com/40272060/123474985-c75f0200-d5c8-11eb-93a0-536952ff26ed.png">